vtex-apps/checkout
# Retrieve settings from dependencies and merge with own settings

#### What is the purpose of this pull request?
Merge all checkout settings.

#### What problem is this solving?
`vtex.checkout` merges all checkout settings from apps that depend on it (located in the corresponding `dist/vtex.checkout/checkout.json` files) with its own settings.

#### How should this be manually tested?
Link apps that depend on `vtex.checkout` and have a corresponding `checkout/checkout.json` file and observe that their contents are merged (with priority given to settings defined by `vtex.checkout`, should properties have the same name).

#### Types of changes
